gapic-generator-java/src/test/java/com/google/api/generator/spring/SpringWriterTest.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.spring;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.api.generator.gapic.composer.common.TestProtoLoader;
+import com.google.api.generator.gapic.model.GapicContext;
+import com.google.api.generator.test.framework.Assert;
+import com.google.api.generator.test.framework.Utils;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SpringWriterTest {
+  private GapicContext context;
+
+  @Before
+  public void setUp() {
+    this.context = TestProtoLoader.instance().parseShowcaseEcho();
```

Comment:
We may need to migrate these test protos as well, as they would not exist in the resources anymore.

Reply:
Yes - thank you for noting! The two [test protos](https://github.com/googleapis/gapic-generator-java/tree/main/gapic-generator-java/src/test/proto) that need to be copied are `echo.proto` and `wicked.proto`.

Reply:
Hmm this might apply to the usages of `TestProtoLoader` and `GrpcRestTestProtoLoader` as well - @blakeli0 do we publish (or intend to publish) a test jar for gapic-generator-java to maven central? If not, we can also migrate an adapted version of these loaders as spring generator test utils, along with the copied protos.